src/Arcus.Security.Tests.Integration/KeyVault/KeyVaultSecretProviderTests.cs

```diff
@@ -246,9 +246,37 @@ public async Task KeyVaultSecretProvider_WithDefaultManagedServiceIdentity_GetSe
                 await Assert.ThrowsAsync<SecretNotFoundException>(async () =>
                 {
                     // Act
-                    await keyVaultSecretProvider.GetSecretAsync(notExistingKeyName);
+                    await keyVaultSecretProvider.GetSecretAsync(notExistingSecretName);
                 });
             }
         }
+
+        [Fact]
+        public async Task KeyVaultSecretProvider_StoreSecret_Succeeds()
```

Comment:
Would be good to have this as an integration test with caching as well, is that ok for you?

Reply:
Ah, yes, that would be good. Thanks.

Reply:
Yes, added a dedicated file for this (different class name/and target to test). Resolving.